BDArmory/UI/LoadedVesselSwitcher.cs

```diff
@@ -1021,168 +1052,213 @@ private void UpdateCamera()
                                 if (wms.Current != null && wms.Current.vessel != null)
                                 {
                                     float vesselScore = 1000;
-                                    float targetDistance = 5000 + (float)(rng.NextDouble() * 100.0);
-                                    float crashTime = 30;
-                                    string vesselName = v.Current.GetName();
-                                    // avoid lingering on dying things
-                                    bool recentlyDamaged = false;
-                                    bool recentlyLanded = false;
+                                    if (currentMode == 1)                                    {
```

Comment:
A minor optimisation, but `switch` instead of multiple `if` statements would be more appropriate here.

Reply:
Implemented